### ISSUE TITLE
Sourcecode is not included / shown; even with `--sourcecode`

### ISSUE THREAD

**Issue by mvriel** [phpDocumentor/phpDocumentor#2671]
When I wanted to write a unit test for the Sourcecode writer (and found out that there is some other improvements to be made there) I discovered that the source code inclusion / generation is not present. It is not used in any template.

## Expected behavior

When I pass `--sourcecode`; that the generated documentation includes the sourcecode of my project and there are links to it in the generated documentation's description of elements

## Actual behavior

No sourcecode is included; and no links are generated

**Comment by LucasKovacs** [comment]
Any fix for this? or temporary fix?

**Comment by jaapio** [comment]
Just noticed that the source writer is not refactored when we introduced the flysystem destinations. This needs to be changed.

**Comment by jaapio** [comment]
Some more info about this issue, it seems that the writer is not executed. Since we are doing some large rework in the way we render documentationsets I think we need to postpone this. 
Blocked by #2837

**Comment by jaapio** [comment]
https://github.com/phpDocumentor/phpDocumentor/pull/2919 fixed
